test/filters/test_crop_hull.cpp

```diff
@@ -356,6 +356,31 @@ TYPED_TEST (PCLCropHullTestFixture, test_cloud_filtering)
 }
 
 
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+TYPED_TEST (PCLCropHullTestFixture, test_keep_organized)
+{
+  for (auto & entry : this->data_)
+  {
+    auto & crop_hull_filter = entry.first;
+    crop_hull_filter.setKeepOrganized(true);
+    crop_hull_filter.setUserFilterValue(-10.);
+    const pcl::PointXYZ defaultPoint(-10., -10., -10.);
+    for (TestData const & test_data : entry.second)
+    {
+      crop_hull_filter.setInputCloud(test_data.input_cloud_);
+      pcl::PointCloud<pcl::PointXYZ> filteredCloud;
+      crop_hull_filter.filter(filteredCloud);
+      ASSERT_EQ (test_data.input_cloud_->size(), filteredCloud.size());
+      for (pcl::index_t i = 0; i < pcl::index_t(test_data.input_cloud_->size()); ++i)
```

Comment:
If test_data.input_cloud_ isn't an index, then don't use index_t for i, and just use size_t.

Reply:
applied the suggestion